apps/dashboard/components/logs/checkbox/filter-checkbox.tsx

```
@@ -71,39 +195,42 @@ export const FilterCheckbox = <TCheckbox extends BaseCheckboxOption, TFilter ext
-          </label>
-        </div>
+            >
+              <Checkbox
+                checked={checkboxes.every((checkbox) => checkbox.checked)}
+                className="size-4 rounded border-gray-4 [&_svg]:size-3"
+                onClick={handleSelectAll}
+              />
+              <span className="text-xs text-accent-12">
+                {checkboxes.every((checkbox) => checkbox.checked) ? "Unselect All" : "Select All"}
+              </span>
+            </label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)